src/lib/API/api.js

```diff
@@ -244,19 +242,26 @@ class API {
   /**
    * `/verify/facerecognition` post api call
    * @param {any} payload
-   * @param {string} provider
+   * @param {string} enrollmentIdentifier
+   * @param {any} axiosConfig
    */
-  async performFaceVerification(
-    payload: any,
-    provider: string = 'zoom',
-    axiosConfig: any = {}
-  ): Promise<$AxiosXHR<any>> {
+  performFaceVerification(payload: any, enrollmentIdentifier: string, axiosConfig: any = {}): Promise<$AxiosXHR<any>> {
     const { client } = this
-    const endpoint = `/verify/facerecognition/${provider}`
+    const endpoint = `/verify/face/${enrollmentIdentifier.toLowerCase()}`
```

Comment:
why .toLowerCase() ?
for encode url string you should use encodeURIComponent

Reply:
I need to cast the string to lowercase, as it does the zoom in itself. This is necessary to determine duplicates.

Reply:
let's move it to the server (ZoomProvider)
here using just encodeURIComponent